code/src/components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import moment from "moment"
+import { useSelector } from 'react-redux'
+import "./Header.css"
+
+export const Header = () => {
+
+  const completedTasks = useSelector((store) => store.tasks.items)
+
+  return (
+    <div className="header-container">
+      <div class="header">
+        <span class="title">Your todo list</span>
+        <span className="completed-task">{completedTasks.length} completed tasks</span>
+      </div>
+      <div class="header-footer">
+        <span className="moment">{moment().format("MMMM Do")}</span>
+      </div>
+      <button
+        class="clear-btn"
+        type="button"
+      >
+        Clear all tasks
+        </button>
```

Comment:
This button is not doing anything? 
Maybe you ran out of time? If so, better to comment out this button so it doesn't feel broken.